bin/mutatex

```diff
@@ -274,8 +274,10 @@ else:
     # Run repair runs
     log.info("Running repair runs")
     repair_outcome = parallel_foldx_run(repair_runs, np=args.np)
+    log.info(f"{repair_outcome}")
 
     if False in list(zip(*repair_outcome))[1]:
+        #this needs to be te case when unrecognized molecules are found.
```

Comment:
typo (te/the)

Reply:
I have changed the wording of the comment a little.